### DISCUSSION TITLE
Ability to load scenes by path.

### DISCUSSION THREAD

**Discussion by FirstGearGames** [FirstGearGames/FishNet#188] [Feature Request]
Requested by Novalty.

> FishNet scene manager currently assumes scenes are referenced by name.

>If you have 2 scenes in different folders with the same name and pass the path into the SceneManager, internally the path gets disregarded and the wrong scene can get loaded.

>The solution is to ensure scenes have distinct names in the project but it would be nice to not have to worry.

**Comment by TomWright** [comment]
Novalty here...

I can give more information or raise a PR on this if it's helpful.

**Comment by FirstGearGames** [comment]
A PR would get it in next update. It will have to be a low priority for me
at this time otherwise.

**Reply by TomWright** [reply]
I've raised a PR here: [#205](https://github.com/FirstGearGames/FishNet/pull/205)

**Comment by FirstGearGames** [comment]
Thanks for the PR. I replied to it but to also follow up here...

I do not want to introduce new content until after 3.1.1 is released, which is planned today. This does not mean the PR will make it next update, rather that I can evaluate it after such. But also, the naming changes will need to be reverted in your PR first.

**Reply by TomWright** [reply]
Sure thing. I'm busy right now but will update it when I can

**Reply by FirstGearGames** [reply]
If you do update, be sure to update under 3.2 please and thank you.
